### ISSUE TITLE
sct_compute_snr error for input image not in RPI orientation

### ISSUE THREAD

**Issue by stephaniealley** [neuropoly/spinalcordtoolbox#1688]
### Description

The function crashes for an input image that is not in RPI orientation. It appears to be working correctly for input images that are already in RPI orientation.

### Steps to Reproduce

Data: sct_testing/issues/20180423_issue1688

Run: 
```
sct_compute_snr -i t2s_concat.nii.gz -m gre-me_1_seg.nii.gz -vol 0,1
```

Error:
```Spinal Cord Toolbox (master/caf11f46d92fe8765490a53325acbf0e2b36b8f1)
Running /Users/stall_admin/spinalcordtoolbox/scripts/sct_compute_snr.py -i t2s_concat.nii.gz -m gre-me_1_seg.nii.gz -vol 0,1

Create temporary folder (/var/folders/rl/d6n7m_vn3b5741zs46zym6sr0000gt/T/sct-180423124626-_DMgIp)...
Get dimensions of data...
320 x 320 x 32 x 2

Create temporary folder (/var/folders/rl/d6n7m_vn3b5741zs46zym6sr0000gt/T/sct-180423124626-1-wOpcRA)...
Split along T dimension...
Change orientation...
Merge file back...
rm -rf /var/folders/rl/d6n7m_vn3b5741zs46zym6sr0000gt/T/sct-180423124626-1-wOpcRA
rm t2s_concat_RPI.nii.gz
Traceback (most recent call last):
  File "/Users/stall_admin/spinalcordtoolbox/scripts/sct_compute_snr.py", line 190, in <module>
    main()
  File "/Users/stall_admin/spinalcordtoolbox/scripts/sct_compute_snr.py", line 116, in main
    input_im_rpi = orientation(input_im, ori='RPI', set=True, fname_out=os.path.join(path_tmp, "input_RPI.nii"))
  File "/Users/stall_admin/spinalcordtoolbox/scripts/sct_image.py", line 652, in orientation
    sct.rm(im.file_name + '_' + ori + im.ext)
  File "/Users/stall_admin/spinalcordtoolbox/scripts/sct_utils.py", line 383, in rm
    os.remove(path)
OSError: [Errno 2] No such file or directory: 't2s_concat_RPI.nii.gz'
```

### State of spinalcordtoolbox

```
master/caf11f46d92fe8765490a53325acbf0e2b36b8f1
```

**Comment by stephaniealley** [comment]
@zougloub it looks like sct_compute_snr was modified in PR #1618 to add an -r argument. Maybe this is related?

**Comment by zougloub** [comment]
Fixed by  #1705
